File: src/main/java/io/github/chaosdave34/kitpvp/kits/impl/elytra/KnockerKit.java
package io.github.chaosdave34.kitpvp.kits.impl.elytra;

import io.github.chaosdave34.kitpvp.kits.ElytraKit;
import org.bukkit.Material;
import org.bukkit.enchantments.Enchantment;
import org.bukkit.inventory.ItemStack;

public class KnockerKit extends ElytraKit {
    public KnockerKit() {
        super("elytra_knocker", "Knocker", Material.STICK);
    }

    @Override
    public ItemStack[] getInventoryContent() {
        ItemStack bow = new ItemStack(Material.BOW);
        bow.addEnchantment(Enchantment.ARROW_INFINITE,1 );
        bow.addEnchantment(Enchantment.ARROW_DAMAGE,  3);
        bow.addUnsafeEnchantment(Enchantment.ARROW_KNOCKBACK, 10);

        return new ItemStack[]{
                bow,
        };
    }
}

File: src/main/kotlin/io/github/chaosdave34/kitpvp/kits/impl/elytra/KnockerKit.kt
package io.github.chaosdave34.kitpvp.kits.impl.elytra

import io.github.chaosdave34.kitpvp.kits.ElytraKit
import org.bukkit.Material
import org.bukkit.enchantments.Enchantment
import org.bukkit.inventory.ItemStack

class KnockerKit : ElytraKit("elytra_knocker", "Knocker", Material.STICK) {

    override fun getInventoryContent(): Array<ItemStack?> {
        val bow = ItemStack.of(Material.BOW)
        bow.addEnchantment(Enchantment.INFINITY, 1)
        bow.addEnchantment(Enchantment.POWER, 3)
        bow.addUnsafeEnchantment(Enchantment.PUNCH, 10)
        bow.addUnsafeEnchantment(Enchantment.KNOCKBACK, 10)

        return arrayOf(
            bow,
        )
    }
}